src/specialcharactersui.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module special-characters/specialcharactersui
+ */
+
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+import { createDropdown } from '@ckeditor/ckeditor5-ui/src/dropdown/utils';
+import specialCharactersIcon from '../theme/icons/specialcharacters.svg';
+import InsertSpecialCharacterCommand from './insertspecialcharactercommand';
+import CharacterGridView from './ui/charactergridview';
+import SpecialCharactersNavigationView from './ui/specialcharactersnavigationview';
+
+/**
+ * The special characters UI plugin.
+ *
+ * Introduces the `'specialCharacters'` dropdown.
+ *
+ * @extends module:core/plugin~Plugin
+ */
+export default class SpecialCharactersUI extends Plugin {
+	/**
+	 * @inheritDoc
+	 */
+	static get pluginName() {
+		return 'SpecialCharactersUI';
+	}
+
+	init() {
+		const editor = this.editor;
+		const t = editor.t;
+		const specialCharsPlugin = editor.plugins.get( 'SpecialCharacters' );
+		const command = new InsertSpecialCharacterCommand( editor );
```

Comment:
```suggestion
		const command = new InsertSpecialCharacterCommand( editor );
		const command = editor.commands.get( 'insertSpecialCharacter' );
```

and:

```js
editor.commands.add( 'specialCharacters', command );
```

should be removed.

The command is being registered in the editing part of the feature now.

Reply:
Thnx @pomek addressed this in bc5f58928401b590659ce629ccd4e199f6b12ccd